al-sufi-arabic/description.md

```diff
@@ -0,0 +1,160 @@
+
+The sky culture stick figures of the constellations are the images in the Ulugh Beg manuscript of the **The Book of the Stars**, extracted, and adapted for Stellarium by Mr. Kutaibaa Akraa [^3].
+
+The single source of the sky culture is the **The Book of the Stars** and the aim is to give a faithful representation of the book content.
+
+## References
+1. IBN QUTAYBA, AD-DINAWARI, ABŪ MUḤAMMAD ‘ABDULLAH B. (died ~ 889 AD), KITĀBU'L-ANWĀ’ (on meteorolgy of the Arabs), (Arabic print of the original book, DAIRUT'L-MA‘ARIFI'L-OSMANIA, Hydarabad, India, 1956 AD)
+أبو محمد عبد الله بن مسلم ابن قتيبة الدينوري (توفي 276 هـ)، كتاب الأنواء في مواسم العرب، داءرة المعارف العثمانية، حيدر أباد، الهند، 1375 هـ
+2. Al-Marzoqi, Abū Ali Aḥmad b. Muḥammad b. Al-Ḥassan (died 1030 AD), Al-Azminah wal'Amkinah (Times and Places), (Arabic print of the original book in 2002, World of Books, Beirut, Lebanon).
+الإمام أبو علي أحمد بن محمد بن الحسن المرزوقي (توفي سنة 421 هـ)، الأزمنة والأمكنة، تحقيق د. محمد نايف الدليمي، عالم الكتب، بيروت، لبنان، 1422 هـ.
+3. Al- Ṯaqafī, Abdu'Allah b. Ḥussayn b. Aṣim (died ~ 1012 AD), Al-Anwā‘ wal'Azminah (Meteorology and times), (Arabic print of the original book in 1996, Dar Al-Jeel, Beirut, Lebanon).
+عبد الله بن حسين بن عاصم الثقفي (توفي 403 هـ)، الأنواء والأزمنة، تحقيق: د. نوري حمودي القيسي، د. محمد نايف الدليمي، دار الجيل، بيروت، لبنان، 1416 ه.
+4. [Book of the Stars: Critical edition with commentary by Khalid AlAjaji, digital edition, 2021 AD](https://drive.google.com/drive/folders/1s6JXzftwjMQ5rgZoGE3718EtBLBZtjzr?usp=sharing)
+5. Al-farāhīdī, Abu Abdul'Raḥmān Al-ḫalīl b. Aḥmad (died ~ 786 AD), Kitab Al-cayn (The book of Arabic letter العين), (Arabic print of the original book, 1980 – 1985).
+أبو عبد الرحمن الخليل بن أحمد الفراهيدي (توفي 170 ه)، كتاب العين، 1400 – 1405 ه.
+6. Instructions Nautiques et Routiers Arabes et Portugais , I , Ibn Majid, Gabriel Ferrand, 1921 -1923.
+7. Instructions Nautiques et Routiers Arabes et Portugais , II, Sulayman Al-Mahri et Ibn Majid, Gabriel Ferrand, 1925.
+8. Paul Kunitzsch,Untersuchungen Zur Sternnomenklatur Der Araber, Otto Harrassowitz ·Wiesbaden, 1961
+9. The Arabs and the Stars, Paul Kunitzsch,The Astronomer Abu'L-Husayn Al-Şūfī And His Book on The Constellations, XI, Northampton, England: Variorum Reprints, 1989.
+10. Ptolemy’s ALMAGEST, G. J. Toomer, Princeton University Press, 2nd ed., 1998.
+
+## Author
+Khalid Al-Ajaji: turath.alfalak@gmail.com
+## License
+Illustrations: Public domain, Source: [Bibliothèque nationale de France. Département des Manuscrits. Arabe 5036](https://gallica.bnf.fr/ark:/12148/btv1b60006156.r=.langEN).
+
+Text and data: Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0). The author wishes that this work be cited according to the common rules.
```

Comment:
It would be easier for me if you remove the NC part, otherwise the data won't be included in the base ubuntu package. Also for the mobile version you'll need to grant a special permission for inclusion..

Reply:
OK,
How would you prefer the special permission be phrased?